docs/xls-73d/index.md

```diff
@@ -0,0 +1,32 @@
+# AMM Behavior Changes
+
+{% partial file="/snippets/_ammclawback-disclaimer.md" /%}
+
+The `AMMClawback` amendment enables token issuers to claw back tokens from wallets that have deposited into AMM pools, ensuring regulatory compliance. This amendment also changes the behavior of the `AMMCreate` and `AMMDeposit` transactions.
+
+
+## AMMCreate Changes
+
+Currently, the `AMMCreate` transaction is blocked if you try to create an AMM, and the issuer of one of the paired assets has enabled clawback support.
```

Comment:
maybe more specific:

Currently, the `AMMCreate` transaction is prohibited if either of the AMM pool token's issuer has enabled `lsfAllowTrustLineClawback`.